aliases.json

```diff
@@ -0,0 +1,12 @@
+{
+  "telemetry": {
+    "saved-session": {
+      "src-namespace": "telemetry",
+      "src-doctype": "main"
```

Comment:
It might be worth either dropping the `src-` prefix (simply `doctype`) or to avoid the abbreviation e.g. `source-doctype`.